src/app/services/notification.service.spec.ts

```diff
@@ -0,0 +1,66 @@
+
+import { TestBed } from '@angular/core/testing';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { NotificationStatus } from '@app/types/notification';
+import { NotificationService } from './notification.service';
+
+describe('Notification service', () => {
+  let service: NotificationService;
+  const snackBar = {
+    open:  jest.fn()
+  };
+
+  beforeEach(() => {
+    service = TestBed.configureTestingModule({
+      providers: [
+        NotificationService,
+        {provide: MatSnackBar, useValue: snackBar}
+      ]
+    }).inject(NotificationService);
+  });
+
+  it('should create Notification service', () => {
+    expect(service).toBeTruthy();
+  });
+  
+  it('should open a snackBar with success status', () => {
+    const status = (notificationStatus: NotificationStatus) => {
+      return {
+        duration: 5000,
+        horizontalPosition: 'end',
+        panelClass: notificationStatus
+      }; 
+    };
```

Comment:
Why creating a function for open ? Another solution could have been:
```suggestion
    const status = {
      duration: 5000,
      horizontalPosition: 'end',
      panelClass: 'success'
    }; 
```

Reply:
It was for avoiding to create three objects for each status.